contentgrid-spring-data-rest/src/test/java/com/contentgrid/spring/test/fixture/invoicing/InvoicingApplicationTests.java

```diff
@@ -888,8 +888,11 @@ void getInvoiceContent() throws Exception {
                         .andExpect(status().isOk())
                         .andExpect(content().contentType(MIMETYPE_PLAINTEXT_UTF8))
                         .andExpect(content().string(EXT_ASCII_TEXT))
+                        ;
+                        /* This assertion is changed in SB3; and is technically incorrect
+                        (it should be `Content-Disposition: attachment` or `Content-Disposition: inline` with a filename, never `form-data`)
                         .andExpect(headers().string("Content-Disposition",
-                                is("form-data; name=\"attachment\"; filename*=UTF-8''%s".formatted(encodedFilename))));
```

Comment:
so the assertion was correct before, but a regression in spring-content ?

Reply:
This is not a regression in spring-content. The assertion itself is now failing because spring adds a `filename=[...]` part as well.

However, the semantics of the assertion are incorrect: 

`content-disposition` set to `form data` is only valid in requests as a form field, never in a response: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition#as_a_header_for_a_multipart_body

In a response, `content-disposition` should be set to either `inline` (show in browser) or `attachment` (perform download). In both cases, the `filename` (and `filename*`) can be set as well.

I am going to send a PR to spring-content to fix this, but first wanted to get this upgrade out of the way